packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyAction.tsx

```diff
@@ -26,6 +26,7 @@ import "./CalculatedPropertyAction.scss";
 import type { CalculatedPropertyType } from "./CalculatedPropertyTable";
 import { useMappingClient } from "./context/MappingClientContext";
 import { useGroupingMappingApiConfig } from "./context/GroupingApiConfigContext";
+import { CalculatedPropertyType as CalcPropertyType } from "@itwin/insights-client";
```

Comment:
Instead of renaming the type to `CalcPropertyType`, leave it as is. Rename the other type in `CalculatedPropertyTable.tsx` to  `ICalculatedPropertyTyped`.
While you are at it. On line 34 in `CalculatedPropertyTable.tsx` 
`setSelectedCalculatedProperty: React.Dispatch<React.SetStateAction<CreateTypeFromInterface<CalculatedPropertyType> | undefined>>;`

The `CreateTypeFromInterface` is redundant. Remove it.

Reply:
Renamed imports as per the suggestion.  Also, ```GroupProperty``` ```CustomCalculations``` files also had the same ```React.Dispatch<React.SetStateAction<CreateTypeFromInterface<...Type> | undefined>>;``` so I deleted ```CreateTypeFromInterface``` there too.

Reply:
For the sake of consistency I also changed all other types made from interfaces to match the ```I...Typed``` naming pattern.